<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCheck</class>
 <widget class="QDialog" name="DialogCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据检查工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>显示全部</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>电芯条码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxCells">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>开始圈数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>结束圈数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>充电起始值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>充电结束值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>放电开始值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>放电结束值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>偏差</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>查看</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
